zuora.gemspec

```
@@ -20,23 +20,24 @@ Gem::Specification.new do |s|
   s.test_files  = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.extra_rdoc_files = ['README.md']
 
-  s.add_development_dependency 'artifice', '~> 0.6.0'
-  s.add_development_dependency 'bigdecimal', '< 2.0.0'
-  s.add_development_dependency 'factory_girl', '~> 2.6.4'
-  s.add_development_dependency 'guard-rspec', '~> 0.6.0'
+  s.add_development_dependency 'appraisal'
+  s.add_development_dependency 'artifice'
+  s.add_development_dependency 'factory_bot'
+  s.add_development_dependency 'guard-rspec'
   s.add_development_dependency 'pry'
+  s.add_development_dependency 'pry-byebug'
   s.add_development_dependency 'pry-nav'
```

I don't think we'll need all 3 of these. As you added `pry-byebug` we can probably just keep that:
```suggestion
  s.add_development_dependency 'pry-byebug'
```

---

Resolved here [5a06bc4](https://github.com/Sage/zuora/pull/18/commits/5a06bc421752cc064b60acf8030181d6bf8b815d)